docs-conceptual/azureadps-2.0/Cannot-find-object.md

```diff
@@ -13,20 +13,23 @@ ms.custom: posh-docs-conceptual
 ms.reviewer: rodejo
 ---
 # I'm looking for a user or a group but PowerShell does not return the object
-Sometimes you are trying to find an object, for instance a group or a user, but PowerShell does not return the object you are looking for. However, you are sure it exists, perhaps because it shows up in the portal, or perhaps because you just created it yourself. 
+
+Sometimes you are trying to find an object, for instance a group or a user, but PowerShell does not return the object you are looking for. However, you are sure it exists, perhaps because it shows up in the portal, or because you just created it yourself.
```

Comment:
```suggestion
Sometimes you're trying to find an object--for instance, a group or a user--but PowerShell doesn't return the object you're looking for. However, you're sure it exists because it shows up in the portal or because you just created the object.
```